blackbox/docs/administration/hba.txt

```diff
@@ -0,0 +1,108 @@
+.. _administration_hba:
+
+===============================
+Host Based Authentication (HBA)
+===============================
+
+This section explains how to configure CrateDB client connection and authentication.
+
+By default, the optional setting ``auth.host_based`` is absent and therefore host based
+authentication is disabled. In this instance, the CrateDB cluster allows any
+unauthenticated connections.
+
+To allow authenticated access to CrateDB from specific hosts, you need to configure
+the ``auth.host_based`` setting in the ``crate.yml`` file.
+
+.. NOTE::
+
+   For the authentication feature to work properly, ``auth.host_based`` setting
```

Comment:
would it probably make sense to create a general note in the configuration for cluster wide settings and link from here and also from other settings which behave the same way?